docs/react-native/upgrading.md

```diff
@@ -81,6 +82,47 @@ APIs related to moments should be removed from your code.
 Any place that you were previously instrumenting your app's performance using Moments can now be done using Performance
 Tracing, please refer to [this guide](/react-native/features/traces/) for more information.
 
+## Troubleshooting
+
+The Embrace React Native SDK implements in the shadows the Native version of our product for Android and iOS, neeading at least a sort of minimum versions required to work as expected ([more details here](/react-native/integration/))
+
+## Android
+
+The minimum version of AGP required for the Embrace Swazzler to work as expected is `7.4.2`. If an older version is used for building the React Native Android application it will end successfully but the SDK won't be able to initialize properly, getting the following error in runtime even when everything is configured as expected:
```

Comment:
```suggestion
The minimum version of AGP required for the Embrace Swazzler to work as expected is `7.4.2`. If an older version is used for building the React Native Android application it will still build successfully but the SDK won't be able to initialize properly, getting the following error in runtime even when everything is configured as expected:
```